k8s/ci/test-script.sh

```diff
@@ -181,7 +181,50 @@ do
 waitForDeployment ${target} openebs
 done
 
+echo "-------------- Verifying the existence of udev inside the cstor pool container--------"
+cstor_pool_pods=$(kubectl get pods -n openebs -l app=cstor-pool -o jsonpath="{range .items[*]}{@.metadata.name}:{end}")
+rc=$?
+if [ $rc != 0 ]; then
+	echo "Error occured while getting the cstor pool pod names; exit code: $rc"
+	exit $rc
+fi
+
+for pool_pod in $(echo "$cstor_pool_pods" | tr ":" " "); do
+
+	echo "======================================="
+	echo "Running lsblk command inside the cstor pool pod: $pool_pod to get device names"
+	device_list=$(kubectl exec -it -n openebs "$pool_pod" -c cstor-pool -- lsblk --noheadings --list)
+	echo "Device list $device_list"
+
+	############### lsblk --noheadings --list #######################
+	##      sdb     8:16   0   10G  0 disk                         ##
+	##	sdb9    8:25   0    8M  0 part                         ##
+	##	sdb1    8:17   0   10G  0 part                         ##
+	##	sda     8:0    0  100G  0 disk                         ##
+	##	sda14   8:14   0    4M  0 part                         ##
+	##	sda15   8:15   0  106M  0 part                         ##
+	##	sda1    8:1    0 99.9G  0 part /var/openebs/sparse     ##
+	#################################################################
+
+	## Fetching device name from above output(first row and first column)
+	device_name=$(echo "$device_list" | awk 'NR==1{print $1}')
```

Comment:
picking any device seems to be OK, it would be good ifwe pick a device which cstor is using. But this is also fine.

Reply:
Since we are running in Travis we only create disk-based pools.

Reply:
OK, can you add  `grep disk` to filter disks only.

`fd0      2:0    1     4K  0 disk 
loop1    7:1    0  54.4M  1 loop /snap/core18/1066
loop2    7:2    0  42.8M  1 loop /snap/gtk-common-themes/1313
loop3    7:3    0 149.9M  1 loop /snap/gnome-3-28-1804/67
loop4    7:4    0     4M  1 loop /snap/gnome-calculator/406
loop5    7:5    0  14.8M  1 loop /snap/gnome-characters/296
loop6    7:6    0  1008K  1 loop /snap/gnome-logs/61
loop7    7:7    0   3.7M  1 loop /snap/gnome-system-monitor/100
loop8    7:8    0  91.3M  1 loop /snap/core/8592
loop9    7:9    0  54.7M  1 loop /snap/core18/1668
loop10   7:10   0   956K  1 loop /snap/gnome-logs/81
loop11   7:11   0  14.8M  1 loop /snap/gnome-characters/399
loop12   7:12   0   3.7M  1 loop /snap/gnome-system-monitor/127
loop13   7:13   0   4.2M  1 loop /snap/gnome-calculator/544
loop14   7:14   0  44.9M  1 loop /snap/gtk-common-themes/1440
loop15   7:15   0 160.2M  1 loop /snap/gnome-3-28-1804/116
loop16   7:16   0  91.4M  1 loop /snap/core/8689
sda      8:0    0   100G  0 disk 
└─sda1   8:1    0   100G  0 part /
sr0     11:0    1  1024M  0 rom  
`

Reply:
Done!! Missed some how.